VMEncryption/main/CryptMountConfigUtil.py

```diff
@@ -610,3 +617,50 @@ def restore_mount_info(self, mount_point_or_mapper_name):
             f.write('\n')
 
         self.logger.log("fstab updated successfully")
+
+    # All encrypted devices are unlocked before this function is called
+    def migrate_crypt_items(self):
+        with open('/etc/fstab', 'r') as f:
+            lines = f.readlines()
+        
+        if not self.is_bek_in_fstab_file(lines):
+            self.logger.log("BEK volume not detected in fstab. Adding it now.")
+            lines.append(self.get_fstab_bek_line())
+            self.add_bek_to_default_cryptdisks()
+            with open('/etc/fstab', 'w') as f:
+                f.writelines(lines)
+
+        crypt_items = self.get_crypt_items()
+
+        for crypt_item in crypt_items:
+            self.logger.log("Migrating crypt item: {0}".format(crypt_item))
+            if crypt_item.mount_point == "/" or CommonVariables.osmapper_name == crypt_item.mapper_name:
+                self.logger.log("Skipping OS disk")
+                continue
+
+            if crypt_item.mapper_name and crypt_item.mount_point and crypt_item.mount_point != "None":
+                self.logger.log(msg="Checking if device for mapper name: {0} has valid filesystem".format(crypt_item.mapper_name), level=CommonVariables.InfoLevel)
+                try:
+                    fstype = self.disk_util.get_device_items_property(crypt_item.mapper_name, 'FSTYPE')
+                    if fstype not in ['xfs', 'ext2', 'ext3', 'ext4']:
```

Comment:
Use the list already available in CommonVariables.format_supported_file_systems

Reply:
Done.